epix/detectors.py

```diff
@@ -0,0 +1,79 @@
+import numba
+import epix
+
+# Fixed detector dimensions of XENONnT:
+# See also: https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenonnt:analysis:coordinate_system
+# #coordinate_system_of_xenonnt
+xenonnt_sensitive_volume_radius = 66.4  # cm
+xenonnt_z_gate_mesh = 0.  # bottom of the gate electrode
+xenonnt_z_top_pmts = 7.3936  # cm
+xenonnt_z_cathode = -148.6515  # cm ... top of the cathode electrode
+xenonnt_z_bottom_pmts = -154.6555  # cm ... top surface of the bottom PMT window
+xenonnt_z_lxe = 0.416 # cm ... liquid-gas interface
+
+
+def xenonnt_detector():
+    """
+    Default XENONnT TPC with different sensitive volumes.
+
+    The structure for each volume is as follows:
+        key: VolumeName
+        items: keyword arguments for the Sensitive Volume class
+
+    :return: A list of volumes for the default nT detector.
+    """
+    # TODO: Can we do this differently? Ideas are welcome....
+    # Outer edges of the volume of interest. This is needed for a
+    # preselection of interactions inside the desired volumes before
+    # clustering.
+    outer_cylinder = {'max_z': xenonnt_z_top_pmts,
+                      'min_z': xenonnt_z_bottom_pmts,
+                      'max_r': xenonnt_sensitive_volume_radius
+                      }
+
+    volumes = {'TPC': {'vol_id': 1,
+                       'roi': _make_roi_cylinder(xenonnt_z_cathode,
+                                                 xenonnt_z_gate_mesh,
+                                                 xenonnt_sensitive_volume_radius),
+                       'electric_field': 200,
+                       'create_S2': True,
+                       'xe_density': 2.862,
+                       'efield_outside_map': 200,
+                       'to_be_stored': True,
+                       },
+               'BelowCathode': {'vol_id': 1,
```

Comment:
Here you have the same `vol_id` for the TPC and BelowCathode. Should this be changed to `2`?

Reply:
Sí!